torch/onnx/symbolic_opset9.py

```diff
@@ -722,16 +728,44 @@ def prelu(g, self, weight):
 def silu(g, input):
     return g.op("Mul", input, g.op("Sigmoid", input))
 
+
 def mish(g, input):
     return g.op("Mul", input, g.op("Tanh", g.op("Softplus", input)))
 
+
+def op_with_optional_float_cast(g, op_name, self, *args, other_operands=None, opset_before=None, target_float_t="Float", **kwargs):
+    origin_dtype = self.type().scalarType()
+    require_cast = not sym_help._is_fp(self) and (opset_before is None or sym_help._export_onnx_opset_version < opset_before)
+    if other_operands is None:
+        other_operands = []
+
+    if require_cast:
+        warnings.warn(
+            f"{op_name}-{sym_help._export_onnx_opset_version} only support float types."
+            f"Using cast<{target_float_t}>-{op_name}-cast-back instead.")
```

Comment:
```suggestion
            f"Using cast<{target_float_t}>-{op_name}-cast<{origin_dtype}> instead.")
```
Maybe a more descriptive log listing both type names for cast